src/nav/telemetry-data-platform.yml

```diff
@@ -51,9 +51,9 @@ pages:
             path: /docs/data-apis/convert-to-metrics/create-metrics-other-data-types
       - title: Drop data
         path: /docs/data-apis/manage-data/drop-data-using-nerdgraph
-      - title: View data limits
+      - title: Understand New Relic data limits
         path: /docs/data-apis/manage-data/view-system-limits
-      - title: Query data limits
+      - title: Details on limits
```

Comment:
@zuluecho9 OK, I see how you've separated these using new navigation names. Still, the "Details on limits" is specific to queries, not all types of limits, right?